site/content/Reference/_index.md

```diff
@@ -0,0 +1,248 @@
+---
```

Comment:
I wonder if it would be worth breaking this up into individual topic pages - input / output / modifiers / prometheus.

Reply:
You might be right about that, but I think I'm going to leave that for a future revision.